test/Microsoft.DotNet.Internal.Health.Tests/Microsoft.DotNet.Internal.Health.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <PropertyGroup>
     <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <SelfContained>true</SelfContained>
```

Comment:
Why do we need this for tests?

Reply:
Test projects were also throwing the warning, but I think I can just explicitly set the prop to `false` to satisfy the warning of change in default behavior? Can try

Reply:
It worked, set the prop in test projects to `false` and left a comment for why. Thanks for the catch 🙇

Reply:
Alternatively, you can just also make tests target net8.0